front/components/assistant/conversation/actions/VisualizationActionIframe.tsx

```diff
@@ -219,21 +220,40 @@ export function VisualizationActionIframe({
             />
           </div>
           <div className="relative flex h-full w-full shrink-0 items-center justify-center">
-            {codeFullyGenerated && (
+            {codeFullyGenerated && !errored && (
               <div
-                style={{ height: `${contentHeight}px` }}
+                style={{
+                  height: !errored ? `${contentHeight}px` : "100%",
+                  minHeight: !errored ? "96" : undefined,
+                  maxHeight: "60vh",
```

Comment:
✂️

Reply:
No we need this actually

Reply:
It's already below set with tailwind, why do we need to duplicate?

Reply:
Indeed, nvm me 👍